### PR TITLE
Add lynx rfr 5012

### PR DESCRIPTION
**Pull request by heidisu**

### CODE REVIEW

File: device-types/LYNX Technik AG/RFR-5012.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: LYNX Technik AG
+model: RFR 5012
+slug: lynx-technik-ag-rfr-5012
+u_height: 2.0
+is_full_depth: true
+airflow: passive
```
Comment (sjurtf) [inline device-types/LYNX Technik AG/RFR-5012.yml:7]: According to the datasheet the 5012 has integrated cooling with fans.

File: device-types/LYNX Technik AG/RFR-5014.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ model: RFR 5014
 slug: lynx-technik-ag-rfr-5014
 u_height: 2.0
 is_full_depth: true
-airflow: passive
+airflow: front-to-rear
```
Comment (sjurtf) [inline device-types/LYNX Technik AG/RFR-5014.yml:7]: The RFR 5014 has passive cooling.

Comment (heidisu) [reply device-types/LYNX Technik AG/RFR-5014.yml:7]: my bad, updated the wrong device, gah..

Comment (heidisu) [reply device-types/LYNX Technik AG/RFR-5014.yml:7]: although there is air flow accoring to this https://www.lynx-technik.com/uploads/tx_fyproducts/pdf/LYNX_R_FR_5012_5014_Manual_V2.1.pdf (but it's just passive?)

File: device-types/LYNX Technik AG/RFR-5012.yml (hunk 1 above)
Comment (heidisu) [reply device-types/LYNX Technik AG/RFR-5012.yml]: hopefully correct now?

**Review by sjurtf — APPROVED** [review]

**Pull request merged**